src/mixins/destroy.js

```diff
@@ -0,0 +1,18 @@
+export default {
```

Comment:
This is essentially the shared functionality between Object and Application.. moved to a mixin